Gallery/Topography/NCL_topo_1.py

```diff
@@ -0,0 +1,89 @@
+"""
+NCL_topo_1.py
+===============
+This script illustrates the following concepts:
+   - Drawing a topographic map using 1' data
+   - Drawing topographic data using GMT colormap
+   - Using wget to download a dataset
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/topo_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/topo_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import matplotlib.pyplot as plt
+import xarray as xr
+import numpy as np
+import cartopy.crs as ccrs
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+import cmaps
+import wget
+
+import geocat.viz as gv
+
+###############################################################################
+# Read in data:
+
+# Use wget to retrieve and download NOAA elevation data
+url = 'https://www.ngdc.noaa.gov/mgg/global/relief/ETOPO1/data/ice_surface/grid_registered/netcdf/ETOPO1_Ice_g_gmt4.grd.gz'
+file_name = wget.download(url)
```

Comment:
The package `urllib3` can access data from a website, and it already exists in the environment created using the GeoCAT-examples installation instructions. See if you can use that package to retrieve the data rather than using a totally new one. If you can't do that, then we can add `python-wget` to the geocat-examples dependencies

Reply:
I wasn't able to use `urllib3`, but I got the `requests` package to work. I am pretty sure that this package is part of our geocat-examples environment.